jQuery/ajax.js

```diff
@@ -21,6 +33,50 @@ function getUniqueValuesForField(field, citizens) {
 
 function populateSelectWithOptions(select, options) {
   options.forEach(function(option) {
-    $(select).append($("<option>", { value: option, html: option }));
+    $(select).append($('<option>', { value: option, html: option }));
+  });
+}
+
+function gatherParams() {
+  var params = [];
+
+  var characterValue = $('#character-select').val();
+  params.push({field: 'character', value: characterValue});
+
+  var hometownValue = $('#hometown-select').val();
+  params.push({field: 'hometown', value: hometownValue});
+
+  var sizeValue = $('#size-select').val();
+  params.push({field: 'size', value: sizeValue});
+
+  return params;
+}
+
+function filter(params, citizens) {
```

Comment:
Can you just explain to me what this function does in plain english? 

Like, is it looking for a match on a single object in the three-object array we returned with params?